js/binance.js

```diff
@@ -4900,4 +4902,23 @@ module.exports = class binance extends Exchange {
     async addMargin (symbol, amount, params = {}) {
         return await this.modifyMarginHelper (symbol, amount, 1, params);
     }
+
+    async fetchBorrowRate (currency, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'asset': this.safeCurrencyCode (currency),
```

Comment:
This is wrong usage of `safeCurrencyCode` – it should be used on the response-parsing side only. On request-compiling side we should do the following to properly fill exchange-specific currency ids:

```JavaScript
    async fetchBorrowRate (code, params = {}) {
        await this.loadMarkets ();
        const currency = this.currency (code);
        const request = {
            'asset': currency['id'],
            ...
```

If possible, please add these edits throughout this commit.